modules/data-entry/src/main/frontend/src/questionnaire/Subject.jsx

```diff
@@ -224,11 +225,18 @@ function SubjectMember (props) {
 
   let handleTableResponse = (json) => {
     setTableData(json.rows);
+    let groups = {};
+    json.rows.map( (entry, i) => {
+        groups[entry.questionnaire["@name"]] = groups[entry.questionnaire["@name"]] || [];
+        groups[entry.questionnaire["@name"]].push(entry);
```

Comment:
This can be simplified to avoid self assignment `groups[entry.questionnaire["@name"]] = groups[entry.questionnaire["@name"]]`:
```suggestion
        groups[entry.questionnaire["@name"]]?.push(entry) || (groups[entry.questionnaire["@name"]] = [entry]);
```